src/dataExplorer/components/Results.tsx

```diff
@@ -100,13 +113,23 @@ const Results: FC = () => {
   }
 
   const dataExists = result.parsed && Object.entries(result.parsed).length
+  const setVizState = state => {
```

Comment:
wait, what is `state`?

Reply:
'table' | 'graph', enforced on the view assignment

Reply:
maybe we name it something else since state is pretty loaded?